features/ALB003_Alignment-directions.feature

```diff
@@ -0,0 +1,11 @@
+@implementer-agreement
+@ALB
+Feature: ALB003 - Alignment Directions
+
+  Scenario: Agreement on nested directions of IfcAlignment
```

Comment:
I probably don't understand, which meaning of *direction* is chosen here. (In my head, an alignment could have a direction forward/backwards; or perhaps going to the left/right/up/down ...) I'd propose to reword to *Alignment Nesting* or similar.

Reply:
I would think that just 'alignment nesting' will lead to confusions with the scenarios in ALB002 (they are all about alignment nesting). This is very specific for IfcAlignmentHorizontal, IfcAlignmentVertical and IfcAlignmentCant. There is not an overlapping term for these three in IFC, and the best wording I could come up with is 'direction'. Do you maybe know a good overlapping word/term (or a couple) for these three cases?

Reply:
> This is very specific for IfcAlignmentHorizontal, IfcAlignmentVertical and IfcAlignmentCant.

The rule includes `IfcReferent` and `IfcAlignment` as well. Please double check.

> Do you maybe know a good overlapping word/term (or a couple) for these three cases?

Layout. Horizontal / vertical / cant are the three layouts of an alignment.

Reply:
> Layout. Horizontal / vertical / cant are the three layouts of an alignment.
That makes sense, changed the names in https://github.com/buildingSMART/ifc-gherkin-rules/pull/26/commits/1a08989b12b035aa11668d0f57e94ff821f2f078

> The rule includes IfcReferent and IfcAlignment as well. Please double check.
I've removed IfcAlignment from it from now. At the moment, it is not very clear to me what to do with this rule. There was a discussion at https://github.com/buildingSMART/ifc-gherkin-rules/issues/18 with this idea:

```
    alignment layouts must be in one RelNests relationship, with order H-V-C
    referents shall be in another RelNests than the one of the alignment layouts
    other alignment shall be in another RelNests than the one of alignment layouts, and referents
```
I've asked for a confirmation of this. Do you have any thoughs/ideas?

Reply:
> I've asked for a confirmation of this. Do you have any thoughs/ideas?

Good question! I don't have a definite answer and am in no position to give one. This is something that should be derived from a requirement analysis, IMHO. I like the proposed idea, however can't confirm its validity.